webui/react/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,193 @@
+  const [submissions, setSubmissions] = useState<Submission[]>([]);
+  const [canceler] = useState(new AbortController());
+  const loadableAuth = useAuth();
+  const authUser = Loadable.match(loadableAuth.auth, {
+    Loaded: (auth) => auth.user,
+    NotLoaded: () => undefined,
+  });
+
+  type Submission = ExperimentItem & CommandTask;
+
+  const fetchTasks = useCallback(
+    async (user: DetailedUser) => {
+      const [commands, jupyterLabs, shells, tensorboards] = await Promise.all([
+        getCommands({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getJupyterLabs({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getShells({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+        getTensorBoards({
+          orderBy: 'ORDER_BY_DESC',
+          signal: canceler.signal,
+          sortBy: 'SORT_BY_START_TIME',
+          users: [user.id.toString()],
+        }),
+      ]);
+      const newTasks = [...commands, ...jupyterLabs, ...shells, ...tensorboards];
+      setTasks(newTasks);
+    },
+    [canceler],
+  );
+
+  const fetchExperiments = useCallback(async (user: DetailedUser) => {
+    const response = await getExperiments({
+      orderBy: 'ORDER_BY_DESC',
+      sortBy: 'SORT_BY_START_TIME',
+      userIds: [user.id],
+    });
+    setExperiments(response.experiments);
+  }, []);
+
+  const fetchProjects = useCallback(async () => {
+    const projects = await getProjectsByUserActivity({ limit: 12 });
+    setProjects(projects);
+  }, []);
+
+  useEffect(() => {
+    fetchProjects();
+    if (!authUser) return;
+    fetchExperiments(authUser);
+    fetchTasks(authUser);
+  }, [authUser, fetchExperiments, fetchTasks, fetchProjects]);
+
+  useEffect(() => {
+    let submissions: Submission[] = [];
+    submissions = submissions
+      .concat(experiments as Submission[])
+      .concat(tasks as Submission[])
+      .sort((a, b) => dateTimeStringSorter(b.startTime, a.startTime));
+    setSubmissions(submissions);
+  }, [experiments, tasks]);
+
+  return (
+    <Page title="Home">
+      <Section title="Recent projects">
+        <Grid
+          count={projects.length}
+          gap={ShirtSize.Medium}
+          minItemWidth={250}
+          mode={GridMode.ScrollableRow}>
+          {projects.map((project) => (
+            <ProjectCard
+              curUser={authUser}
+              fetchProjects={fetchProjects}
+              key={project.id}
+              project={project}
+            />
+          ))}
+        </Grid>
+      </Section>
+      <Section title="Recently submitted">
+        <Table<Submission>
```

Comment:
just noticed that pagination settings is initialized after visiting other pages
https://user-images.githubusercontent.com/109113616/207959088-9e8ed127-d22b-44f1-a4c2-4b11843be517.mov

Reply:
pagination wasn't even explicitly required in the design, but @alexjdetermined lmk if you think it should be paginated and if these pagination settings should be saved (prefer to avoid the overhead unless it's deemed necessary).